### DISCUSSION TITLE
Presenter mode possible to protect with password?

### DISCUSSION THREAD

**Discussion by ijpatricio** [slidevjs/slidev#749] [Ideas]
Hello,

Is it possible something like this?

```
pnpm run dev --remote --remote-password=ketchup
```

Or maybe instead of having to develop this, to use vite under the hood? [vite proxy](https://stackoverflow.com/questions/70753555/basic-auth-using-vite-proxy) maybe?

[slidev docs](https://sli.dev/custom/config-vite.html#configure-vite) mention configuring vite here, I'm just afraid if I override any sensible defaults....

**Comment by JoeyGrajciar** [comment]
`remote` option accepts password `--remote=<password>`
[Remote restricted access](https://sli.dev/guide/presenter-mode.html#remote-restricted-access)

**Reply by ijpatricio** [reply]
Thank you so much, I think overlooked it!
